tests/php/sync/test_class.jetpack-sync-full.php

```diff
@@ -167,7 +168,51 @@ function test_full_sync_sends_all_terms() {
 		$this->sender->do_full_sync();
 
 		$terms = $this->server_replica_storage->get_terms( 'post_tag' );
-		$this->assertEquals( 11, count( $terms ) );
+		$this->assertEquals( 22, count( $terms ) );
```

Comment:
where is this 22 coming from?

Reply:
its from 11 categories and 11 tags. Updated with a comment.

Reply:
Let's use a self describing constant instead, e.g. "NUMBER_OF_CATEGORIES_AND_TAGS" so that we're not dealing with "magic numbers".

Reply:
I agree. Otherwise, it seemed to work as expected, so with that change, let's land it.